simulator/src/main/java/peersim/kademlia/KademliaNode.java

```diff
@@ -14,6 +14,7 @@ public class KademliaNode implements Comparable<KademliaNode> {
   /** all the topics the node registers for */
   String myTopic = "";
 
+  private boolean server;
```

Comment:
maybe let's call it `is_server` for consistency with `is_evil`?

Reply:
Ok